components/core/src/clp/regex_utils/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+set(
+        REGEX_UTILS_HEADER_LIST
+        "ErrorCode.hpp"
+        "RegexToWildcardTranslatorConfig.hpp"
+        "constants.hpp"
+        "regex_translation_utils.hpp"
+)
+add_library(
+        regex_utils
+        regex_translation_utils.cpp
+        ErrorCode.cpp
+        ${REGEX_UTILS_HEADER_LIST}
+)
+add_library(clp::regex_utils ALIAS regex_utils)
+target_include_directories(regex_utils
+        PUBLIC
+        ../
```

Comment:
I'm not quite sure. After setting this to private, the unit tests can still call regex_utils stuff correctly. Will come back to this in case the visibility causes an issue in the future

Reply:
How about just switch to `PRIVATE` to be consistent with our other cmake configs